pkg/storkctl/snapshot_test.go

```diff
@@ -21,29 +25,42 @@ import (
 
 type NewTestCommand func(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command
 
-func testSnapshotsCommon(t *testing.T, newCommand NewTestCommand, cmdArgs []string, snapshots *snapv1.VolumeSnapshotList, expected string, errorExpected bool) {
+func testCommon(t *testing.T, newCommand NewTestCommand, cmdArgs []string, obj runtime.Object, expected string, errorExpected bool) {
 	var err error
-
+	typeName := reflect.ValueOf(obj).Elem().Type().Name()
+
+	var addToScheme func(*runtime.Scheme) error
+	var schemeGroupVersion schema.GroupVersion
+	if typeName == "VolumeSnapshotList" {
+		addToScheme = snapv1.AddToScheme
```

Comment:
I think we can call addToSchem() for all the crds. You can do that in an init() function in common_test.go

Reply:
OK, I refactored into an `init()` method:
https://github.com/libopenstorage/stork/pull/195/commits/8e1cc53f8c5e855b96ec05d6a00c79bdc7988651#diff-a7665dbba9e98c4f01c65f3725c7bb93R26